flexget/plugins/filter/series.py

```diff
@@ -421,13 +424,65 @@ def __init__(self, name):
         self.name = name
 
 
+def get_latest_status(episode):
+    """
+    :param episode: Instance of Episode
+    :return: Status string for given episode
+    """
+    status = ''
+    for release in sorted(episode.releases, key=lambda r: r.quality):
+        if not release.downloaded:
+            continue
+        status += release.quality.name
+        if release.proper_count > 0:
+            status += '-proper'
+            if release.proper_count > 1:
+                status += str(release.proper_count)
+        status += ', '
+    return status.rstrip(', ') if status else None
+
+
+def display_series_summary(configured=None, premieres=None, status=None, days=None, session=None):
```

Comment:
Is 'display' really the best way to describe this? It doesn't really display anything.